lib/identity_cors.rb

```diff
@@ -0,0 +1,37 @@
+# frozen_string_literal: true
+
+class IdentityCors
```

Comment:
since it's never instantiated, maybe switch to `module` and use `module_function` so the methods get copied to the class? (ex no need to `def self.method` just `def method`)